rtengine/procparams.cc

```diff
@@ -390,51 +400,51 @@ bool ToneCurveParams::operator !=(const ToneCurveParams& other) const
 RetinexParams::RetinexParams() :
     enabled(false),
     cdcurve{
-        DCT_Linear
```

Comment:
@Desmis @Pandagrapher Obviously `astyle` destroyed the formatting. Would you mind to repair it? Afterwards it's easier to review this file.

Reply:
@Floessie 
I just push a "small" change tp procparams.cc